docs/project/copyright.md

```diff
@@ -1,31 +1,32 @@
 Copyright
 =========
 
-The .NET project copyright is held by ".NET Foundation and Contributors".
+.NET is a member project of the [.NET Foundation](http://www.dotnetfoundation.org/).
 
-The [.NET Foundation](http://www.dotnetfoundation.org/) is an independent organization that encourages open development and collaboration around the .NET ecosystem.
+Source copyright is held by ".NET Foundation and Contributors", while binary distributions (and associated copyright) are provided by a variety of parties.
 
 Source License
 --------------
 
 The .NET project uses multiple licenses for the various project repositories.
 
-- The [MIT License](https://opensource.org/licenses/MIT) is used for [code](https://github.com/dotnet/runtime/).
-- The [Creative Commons Attribution 4.0 International Public License (CC-BY)](https://creativecommons.org/licenses/by/4.0/) is used for [documentation](https://github.com/dotnet/docs/) and [swag](https://github.com/dotnet/swag).
+- The [MIT License](https://opensource.org/licenses/MIT) is used for code, for example [dotnet/runtime](/LICENSE.TXT).
+- Various [Creative Commons](https://creativecommons.org/) licenses are used for non-code assets, like [documentation](https://github.com/dotnet/docs/), [swag](https://github.com/dotnet/swag), and [brand](https://github.com/dotnet/brand) content.
+- The [.NET Foundation Contribution License Agreement (CLA)](https://cla.dotnetfoundation.org) is used to license contributions from contributors.
+
+We exclusively accept code contributions with permissive [OSI-approved licenses](https://opensource.org/licenses), for example [MIT](https://opensource.org/licenses/MIT), [Apache 2](https://opensource.org/licenses/Apache-2.0), and [BSD](https://opensource.org/licenses/BSD-3-Clause).
+
+We do not accept code contributions with [GPL](https://opensource.org/licenses/gpl-license), [creative commons](https://creativecommons.org/) ([CC0](https://opensource.org/faq#cc-zero), [CC BY-SA](https://opensource.stackexchange.com/questions/1717/why-is-cc-by-sa-discouraged-for-code)), or [public domain licenses](https://opensource.org/faq#public-domain).
```

Comment:
For what it's worth, .NET carries some Public Domain code. Fedora hasn't banned that. In fact, we have documented the examples from coreclr and fsharp at https://gitlab.com/fedora/legal/fedora-license-data/-/blob/main/public-domain-text.txt#L38-49

I know that some jurisdictions don't have a public domain, so it might make sense to discourage public domain code in .NET, but I just wanted to call out that we don't need to ban public domain because of Fedora (only).

Reply:
Agreed. I made some updates to better clarify that in https://github.com/dotnet/runtime/pull/80272/commits/b6fa25b4eebea3904772c8b3bd20f5458f283999.